include/dataformats/GeoID.hpp

```diff
@@ -57,6 +65,14 @@ struct GeoID
    */
   uint32_t element_id{ s_invalid_element_id }; // NOLINT(build/unsigned)
 
+  GeoID() {}
```

Comment:
Can I suggest for the defaul constructor to set the values to invalid?

Reply:
It does. Since it's not explicitly initializing the values, the default (invalid) initializers are used.